src/util-file.c

```diff
@@ -149,6 +155,25 @@ void FileForceTrackingEnable(void)
     g_file_force_tracking = 1;
 }
 
+void FileWriteMetaDisable(void)
```

Comment:
why are these meta things globals in util-file.c? They are specific to the file output logger right?